docs/usage/cli.rst

```diff
@@ -117,7 +117,24 @@ Options
 +-------------------------------------+---------------------------+-----------------------------------------------------------------+
 | ``--debug``                         | ``LITESTAR_DEBUG``        | Run the application in debug mode                               |
 +-------------------------------------+---------------------------+-----------------------------------------------------------------+
+| ``--reload-dir``                    | ``LITESTAR_RELOAD_DIRS``  | Specify directories to watch for reload.                        |
```

Comment:
```suggestion
| ``--reload-dir``                    | ``LITESTAR_RELOAD_DIRS``  | Specify comma delimited list of directories to watch for reload.                        |
```

Reply:
but this is not how it works?

Reply:
I suppose it is a little bit ambiguous that the env var behaves different to the `--reload-dir` flag - but the flag behaves the same as uvicorn, so I'd rather keep that behavior. What do you suggest?

Reply:
Also to add, this is why I added the section below this with a description and examples of the flag and env var behavior.